runtime/centrifuge/src/lib.rs

```diff
@@ -1252,8 +1252,8 @@ impl_runtime_apis! {
 			let weight = Executive::try_runtime_upgrade().unwrap();
 			(weight, RuntimeBlockWeights::get().max_block)
 		}
-		fn execute_block_no_check(block: Block) -> Weight {
-			Executive::execute_block_no_check(block)
+		fn execute_block(block: Block, state_root_check: bool, try_state: TryStateSelect) -> Weight;
```

Comment:
TODO: Have to figure out how this is actually used vs. and if we used `execute_block_no_check` in the past.

Reply:
ASAIR the no-check prevents digest checks for block building. Do not think we can anything about the change and hopefully parity has this correctly ^^